docs/src/main/paradox/external-connectors.md

```diff
@@ -1,8 +1,10 @@
 # External Connectors
 
-Connectors that are maintained by other projects.
+[![alpakka]][alpakka-scaladex] Feel free to tag your project with *alpakka* keyword in Scaladex for easier discoverability.
```

Comment:
perhaps we should include a link to the docs that explain how to tag projects in Scaladex?